src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -89,7 +93,7 @@ export default class Toolbar extends React.Component<*> {
                     <ToolbarComponent.Button
                         onClick={this.handleNavigationButtonClick}
                         skin="primary"
-                        icon={NAVIGATION_BUTTON_ICON}
+                        icon={navigationOpen ? 'times' : NAVIGATION_BUTTON_ICON}
```

Comment:
I'd say we remove this and the other constants, instead of mixing hardcoded strings and constants.